software/control/gui_hcs.py

```diff
@@ -609,35 +587,25 @@ def setupMultiWindowLayout(self):
     def makeConnections(self):
         self.streamHandler.signal_new_frame_received.connect(self.liveController.on_new_frame)
         self.streamHandler.packet_image_to_write.connect(self.imageSaver.enqueue)
-        # self.streamHandler.packet_image_for_tracking.connect(self.trackingController.on_new_frame)
-        self.navigationController.xPos.connect(lambda x: self.navigationWidget.label_Xpos.setText("{:.2f}".format(x) + " mm"))
-        self.navigationController.yPos.connect(lambda x: self.navigationWidget.label_Ypos.setText("{:.2f}".format(x) + " mm"))
-        self.navigationController.zPos.connect(lambda x: self.navigationWidget.label_Zpos.setText("{:.2f}".format(x) + " μm"))
 
-        if SHOW_NAVIGATION_BAR:
-            self.navigationController.xPos.connect(self.navigationBarWidget.update_x_position)
-            self.navigationController.yPos.connect(self.navigationBarWidget.update_y_position)
-            self.navigationController.zPos.connect(self.navigationBarWidget.update_z_position)
-
-        if ENABLE_TRACKING:
-            self.navigationController.signal_joystick_button_pressed.connect(self.trackingControlWidget.slot_joystick_button_pressed)
-        else:
-            self.navigationController.signal_joystick_button_pressed.connect(self.autofocusController.autofocus)
+        # TODO(imo): Fix joystick after removal of navigation controller
```

Comment:
This TODO needs to be addressed before merge

Reply:
Currently the joystick buttons are not implemented in the firmware. Previously joystick is connected directly to the microcontroller (arduino). Now it’s joystick -> teensy lc ->  teensy 4.1 -> computer.